packages/@ember/-internals/routing/lib/location/history_location.ts

```diff
@@ -185,11 +180,7 @@ export default class HistoryLocation extends EmberObject implements EmberLocatio
     @return state {Object}
   */
   getState() {
```

Comment:
Can we delete this method instead? Seems like it was only used as an abstraction over the polyfill?

Reply:
done